app/views/shared/_user_fields.html.slim

```diff
@@ -0,0 +1,15 @@
+= simple_form_for @user do |f|
+  - required_new = (action_name == "new")
```

Comment:
There's a better way to do this.

`@user.new_record?` will return true if newly initialized and not persisted, will return false otherwise.